languages/julia/exercises/concept/lasagna/.docs/introduction.md

```diff
@@ -1,8 +1,6 @@
 The entire Julia track will require you to treat your solution like small libraries, i.e. you need to define functions, types etc. which will then be run against a test suite.
 For that reason, we will introduce named functions as the very first concept.
 
-## functions-introduction
-
 ### Defining functions
```

Comment:
What's the correct level here now?

Reply:
I think you can use level 2 headings or more, right @iHiD?